build.gradle.kts

```diff
@@ -4,9 +4,10 @@ val testContainersVersion = "1.17.5"
 val log4jVersion = "2.17.2"
 
 plugins {
+    // gradle-release has to go first, see https://github.com/atlassian-labs/gradle-release/pull/5/commits/0e89b0646d3393e5cf2827240d64bc9bdc2060c1
```

Comment:
Nit: Maybe it's just me, but I find the message of linked commit not completely clear. I don't think that matters too much though.

Reply:
True, the second paragraph is the most relevant part, the first one is only partially talking about what will happen if we don't have the workaround. I should have split that commit into two.